status.inc.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * Get approot.inc.php
+ * Move to a function for allowing a better testing
+ * 
+ * @param string $sAppRootFilename
+ * @throws \Exception
+ */
+function StatusGetAppRoot($sAppRootFilename = 'approot.inc.php') 
+{
+        $sAppRootFile = dirname(__FILE__).'/'.$sAppRootFilename;
+
+        /*
+        * Check that the approot file exists and has the appropriate access rights
+        */
+        if (!file_exists($sAppRootFile) || !is_readable($sAppRootFile)) 
+        {
+            throw new \Exception($sAppRootFile . ' is not readable');
```

Comment:
We usually don't throw / show absolute paths as it would expose the application to a *full path disclosure* security flaw. So we would recommand to change the exceptions accordingly.

```suggestion
            throw new \Exception($sAppRootFilename . ' is not readable');
```